lib/api/apiUtils/bucket/validateSearch.js

```diff
@@ -63,6 +64,11 @@ function _validateTree(whereClause, possibleAttributes) {
 function validateSearchParams(searchParams) {
     let ast;
     try {
+        // allow using 'replicationStatus' as search param to increase
+        // ease of use, pending metadata search rework
+        // eslint-disable-next-line no-param-reassign
+        searchParams = searchParams.replace(
```

Comment:
Unless it has been decided otherwise for good reason, let's try to avoid mixing camelCase and dash-separated as accepted search words for consistency (especially if search is case-insensitive, that I don't know), `replication-status` would be better in this respect.